homeassistant/components/daikin/climate.py

```diff
@@ -81,12 +81,7 @@ def __init__(self, api):
         self._api = api
         self._list = {
             ATTR_OPERATION_MODE: list(HA_STATE_TO_DAIKIN),
-            ATTR_FAN_MODE: list(
-                map(
-                    str.title,
-                    appliance.daikin_values(HA_ATTR_TO_DAIKIN[ATTR_FAN_MODE])
-                )
-            ),
+            ATTR_FAN_MODE: self._api.device.fan_modes,
```

Comment:
This is wrong. We should only expose the fan modes as defined by Home Assistant.

Reply:
You say that there are predefined supported modes?

In this case does the Daikin AC report different modes based on model, I don't think they should be defined in Home Assistant as it is more a library task.

Reply:
@balloob please advise how I should proceed when there are multiple options based on different models. The library handles this atm so HomeAssistant does not have to.

Reply:
@balloob we haven't yet defined allowed fan modes for the climate component. For the fan component we have done this.

So for now we've let integrations define their own fan modes in the climate component but they have to follow the operation modes that home assistant has set.

Can we wait with enforcing fan modes for climate platforms until we've defined what those should be?

Reply:
Sure.

Reply:
So we are good to go?

Reply:
There's no I/O here, right?

Reply:
Nope.